libraries/forms.py

```diff
@@ -608,6 +609,41 @@ def apply_colors(self):
         graph.apply_colors()
         return graph
 
+    def _get_slack_stats(self, prior_version, version):
+        """Returns all slack related stats."""
+        start = prior_version.release_date
```

Comment:
@vinniefalco Should we have a way to filter to specific channels, or are we always going to use all channels?

Reply:
As it is, it is aggregating stats for all channels the bot as access to (which is just general and boost-website atm I believe).

Reply:
only channels starting with "boost"